src/test/tests/databases/silo.py

```diff
@@ -635,4 +635,46 @@ def TurnOffSetsByName(silr, cat, names):
 DeleteAllPlots()
 CloseDatabase(data_path("silo_ghost_test_data/specmix_ucd.pdb"))
 
+#
+# Test the way silo reads in curve data with expressions
+#
+OpenDatabase(data_path("silo_pdb_test_data/multi_rect2d.silo"))
```

Comment:
use ``silo_data_path("multi_rect2d.silo")`` instead of ``data_path(...)``. This is to handle the fact that ``silo.py`` actually gets run twice. Once as ``silo.py`` on Silo's HDF5 driver and once as ``silo_altdriver.py``.